operations/power_management/System_Power_Off_Procedures.md

```diff
@@ -22,7 +22,7 @@ To save management switch configuration settings, refer to [Save Management Netw
 
 ## Power Off System Cabinets
 
-To power off standard rack and liquid-cooled cabinet PDUs, refer to [Power Off Compute and IO Cabinets](Power_Off_Compute_and_IO_Cabinets.md).
+To power off standard rack and liquid-cooled cabinet PDUs, refer to [Power Off Compute Cabinets](Power_Off_Compute_Cabinets.md).
```

Comment:
This link does not work. Maybe you need to delete the old file, Power_Off_Compute_and_IO_Cabinets.md
and have a new file named Power_Off_Compute_Cabinets.md?
Or maybe move it to its new name?